### ISSUE TITLE
Breaks with AVA 0.15.1

### ISSUE THREAD

**Issue by jrolfs** [sheerun/ava-spec#3]
I'm planning on looking into this and hopefully opening a PR, but after upgrading AVA in my project that uses `ava-spec` I'm getting the following error:

```
[my-project]/node_modules/ava/lib/runner.js:101
                if (fn.title) {
                      ^

TypeError: Cannot read property 'title' of null
    at Runner._addTest ([my-project]/node_modules/ava/lib/runner.js:101:9)
    at Runner.<anonymous> ([my-project]/node_modules/ava/lib/runner.js:89:8)
    at Function.wrappedFn ([my-project]/node_modules/option-chain/index.js:36:14)
    at Function.<anonymous> ([my-project]/node_modules/ava-spec/lib/index.js:91:19)
    at Function.wrappedFn ([my-project]/node_modules/option-chain/index.js:36:14)
    at Object.<anonymous> ([my-project]/test/mixins/event-emitter.test.js:10:6)
    at Module._compile (module.js:541:32)
    at extensions.(anonymous function) ([my-project]/node_modules/require-precompiled/index.js:13:11)
    at Object.require.extensions.(anonymous function) [as .js] ([my-project]/node_modules/ava/lib/test-worker.js:91:3)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> ([my-project]/node_modules/ava/lib/test-worker.js:95:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Function.Module.runMain (module.js:575:10)
    at startup (node.js:160:18)
    at node.js:449:3
```

My first guess is possibly some sort of conflict with the Macro support that was introduced? It looks like the tests in this project don't actually run against AVA, maybe we should consider doing so to catch these sorts of things?

**Comment by sheerun** [comment]
Hey,

I cannot reproduce the error, could paste your test file, or even better attach failing integration test in https://github.com/sheerun/ava-spec/blob/master/spec/test.js ?

**Comment by jrolfs** [comment]
Thanks for adding the integration tests! It looks like it is the `beforeEach` is the culprit: #4

**Comment by jrolfs** [comment]
Closing as this was fixed with #4

**Comment by sheerun** [comment]
Released your fix in 1.0.1, could you confirm it works?

**Comment by jrolfs** [comment]
Yep, thanks!
![fixed](http://d.pr/i/1dqXv+)
